### ISSUE TITLE
Use a CI service to publish releases

### ISSUE THREAD

**Issue by fafnirical** [forumone/wp-cfm#91]
When a release is created on GitHub, a release could be created on WP.org. This could also be used to run tests against different WordPress/WP-CLI versions

**Comment by elvismdev** [comment]
Yes, this is currently how we have it set up through Jenkins CI, with the exception that we have no tests to run yet. Once an update is pushed to Github it will be pushed to WP.org as well.
